go.mod

```diff
@@ -19,21 +19,21 @@ require (
 	github.com/pborman/uuid v1.2.0
 	github.com/pelletier/go-toml v1.9.3
 	github.com/prometheus/client_golang v1.11.0
-	github.com/sirupsen/logrus v1.7.0
+	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
-	google.golang.org/grpc v1.38.0
+	google.golang.org/grpc v1.41.0
 	google.golang.org/protobuf v1.26.0
-	k8s.io/api v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
+	k8s.io/api v0.22.3
+	k8s.io/apimachinery v0.22.3
+	k8s.io/client-go v0.22.3
 	k8s.io/cloud-provider v0.22.2
```

Comment:
We should keep the k8s.io packages at the same version. Was it really necessary to upgrade to v0.22.3? This seems to have caused a lot of churn in the vendor directory.

Reply:
Yes, there were certain code changes in apimachinery and client-go which were required. I'll update the other packages to v0.22.3 as well.

Reply:
You should also update the corresponding replace statements to use v0.23.0 as well.

Reply:
I am thinking of only updating the regularly updated pkg under k8s.io to v0.22.3, since there are a lot of packages with older versions and on updating those we might risk the chance of breaking something as create a lot of churn in the vendor directory.

Reply:
FYI- The `k8s.io/kubernetes` package seems to declare dependencies on `v0.0.0` of all the other `k8s.io` packages which requires us to include `replace` statements to override with the correct version to include.

Reply:
Yes, I have already added a couple of pkgs in replace for the same reason.

Reply:
> I am thinking of only updating the regularly updated pkg under k8s.io to v0.22.3, since there are a lot of packages with older versions and on updating those we might risk the chance of breaking something as create a lot of churn in the vendor directory.

It makes more sense to keep the `k8s.io` packages on the same version since that's how they are developed and tested.

Reply:
Done. All the k8s.io pkgs are updated to use v0.22.3